devicetypes/smartthings/child-switch-health.src/child-switch-health.groovy

```diff
@@ -32,6 +32,10 @@ metadata {
 	}
 }
 
+def intalled() {
+	sendEvent(name: "checkInterval", value: 30 * 60, displayed: false, data: [protocol: "zigbee", hubHardwareId: device.hub.hardwareID])
```

Comment:
Add a comment indicating this is set to a default value but should be overridden by the parent.